web3signer/security/lodestar/api-token.txt

```diff
@@ -0,0 +1 @@
+api-token-0x8107edae469565b8cc7cd08addc2bd9a73cd9cfe8127e652caa4c7b5ad50a4bd
```

Comment:
the api-token does not match the value of [DAppNodePackage-Lodestar api-token.txt](https://github.com/dappnode/DAppNodePackage-Lodestar/blob/main/validator/security/api-token.txt), I would suggest we align the api-token to be the same for all packages as it is done for other CLs. Related PR https://github.com/dappnode/DAppNodePackage-Lodestar/pull/36 and https://github.com/dappnode/StakingBrain/pull/196

```suggestion
api-token-0x7fd16fff6453982a5d8bf14617e7823b68cd18ade59985befe64e0a659300e7d
```